### PR TITLE
9 task description

### PR DESCRIPTION
**Pull request by sylvihuynh**
- create a place for the task title and task description
- use accordion to create an expand component

#9

### CODE REVIEW

File: pages/assessment.tsx [hunk 1]
```diff
@@ -24,6 +27,7 @@ const Assessment: NextPage = () => {
   const [currentPage, setCurrentPage] = useState<number>(1);
   const [maxItemsPerPage, setMaxItemsPerPage] = useState<number>(4); //max items set to 4 as default
 
+  const taskDescription: string = 'Variabler med nøkkelordet var er globale, mens varibler med nøkkelordet let har et local scope eller blokk scope som vil si at de kun defineres for deler av koden om de defineres inni en kodeblokk.'
```
Comment (sylvihuynh) [inline pages/assessment.tsx:30]: currently an answer is used as a task description :s

**Comment by sylvihuynh** [comment]
Somehow everything is pushed down to the left when opening the expand component 😕

File: pages/assessment.tsx (hunk 1 above)
Comment (amalieeh) [reply pages/assessment.tsx:30]: Yeah, that'll be fixed in my pr #33 :)

**Review by amalieeh — APPROVED** [review]
Noise! It works really well and looks good! 
Great job!!

**Pull request merged**